packages/esm-appointments-app/src/calendar/monthly/days-of-week.component.tsx

```diff
@@ -19,7 +20,11 @@ interface DaysOfWeekProps {
 
 const DaysOfWeekCard: React.FC<DaysOfWeekProps> = ({ dayOfWeek }) => {
   const { t } = useTranslation();
-  const isToday = dayjs(new Date()).format('ddd').toUpperCase() === dayOfWeek;
+  const todayDate = today(getLocalTimeZone());
+  const isToday =
+    new Intl.DateTimeFormat(getLocale(), { weekday: 'short' })
+      .format(todayDate.toDate(getLocalTimeZone()))
+      .toUpperCase() === dayOfWeek;
```

Comment:
I don't love having the `toUpperCase()` thing here... Maybe we refactor things to that the _caller_ instead passes in the `dayOfWeek` (which should already be translated) and a boolean for whether or not it's today?

Then this just becomes:

```tsx
interface DayOfWeekProps {
  dayOfWeek: string;
  isToday: boolean;
}

const DayOfWeekCard: React.FC<DayOfWeekProps> = ({ dayOfWeek, isToday }) => {
  return (
    <div tabIndex={0} role="button" className={styles.tileContainer}>
      <span className={classNames({ [styles.bold]: isToday })}>{dayOfWeek}</span>
    </div>
  );
}
```